cmd/dockerd/daemon.go

```diff
@@ -450,23 +450,21 @@ func newRouterOptions(ctx context.Context, config *config.Config, d *daemon.Daem
 			Named:    d.ImageNamedByBuildkit,
 		},
 	})
-	if err != nil {
-		return routerOptions{}, err
-	}
 
 	bb, err := buildbackend.NewBackend(d.ImageService(), manager, bk, d.EventsService)
 	if err != nil {
-		return routerOptions{}, errors.Wrap(err, "failed to create buildmanager")
+		return nil, nil, errors.Wrap(err, "failed to create buildmanager")
 	}
+	return bb, sm, nil
```

Comment:
Would it make sense to log here that we finished initialising BuildKit? To make it the counterpart of; 

```go
log.G(ctx).Info("Completed initializing BuildKit")
```

or

```go
log.G(ctx).Info("Finished initializing BuildKit")
```